src/jsonapi.js

```diff
@@ -139,7 +139,22 @@ export const readEndpoint = (endpoint, {
         .then(json => {
           dispatch(apiRead({ endpoint, ...json }));
           onSuccess(json);
-          resolve(json);
+
+          const prevUrl = getPaginationUrl(json, 'prev', apiHost, apiPath);
+          const nextUrl = getPaginationUrl(json, 'next', apiHost, apiPath);
+
+          const getPrevPage = () => dispatch(readEndpoint(prevUrl));
+          const getNextPage = () => dispatch(readEndpoint(nextUrl));
+
+          const resObj = {
```

Comment:
For documentation and clean-code purposes I think it'd be favorable to instantiate a named class to use here.